app/controllers/api/applications_controller.rb

```diff
@@ -11,9 +11,44 @@ def update #move to command
     if @application.update(application_params)
       ActiveRecord::Base.transaction do
```

Comment:
I think we don't want this in a transaction - if saving addresses fails, we still want to try to save employers. Let's take out the transaction and add a rescue to each attempt at a save that handles any errors.